server/src/service/agenda.ts

```diff
@@ -2,7 +2,40 @@ import type { Prisma } from "@prisma/client";
 import * as schema from "biseo-interface/agenda";
 import { prisma } from "@/db/prisma";
 
-export const retrieveAll = async ({}): Promise<schema.RetrieveAllCb | null> => {
+export const retrieveAll = async (): Promise<schema.RetrieveAllCb | null> => {
+  const agenda_dbres = await prisma.agenda.findMany({
```

Comment:
snake case가 아니라 camel case로 변수명을 써주면 좋을 것 같습니다.

```suggestion
  const agendas = await prisma.agenda.findMany({
```